src/main/webapp/aboutus.html

```diff
@@ -42,17 +42,11 @@ <h2>Teammate B Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate C Name</h2>
```

Comment:
noted in other PRs, lets change back to 2 spaces and deal with the mass indent fix later